sample-apps/discount-functions-sample-app/extensions/fixed-product-discount/input.graphql

```diff
@@ -0,0 +1,13 @@
+query Input {
+  merchandiseLines {
+    variant {
+      id
+    }
+  }
```

Comment:
This needs the new Cart API input and new schemas so idk if it's easier for you to redo this from the latest sample `product-discount` 😅